spi/src/main/java/com/flipkart/varadhi/spi/services/PolledMessage.java

```diff
@@ -24,12 +24,6 @@ public interface PolledMessage<O extends Offset> extends Message {
      */
     O getOffset();
 
-    /**
-     * @return the payload of this message.
-     */
-    // TODO: evaluate ByteBuf instead of byte[]. byte[] means heap allocated.
-    byte[] getPayload();
```

Comment:
why did we remove this method?

Reply:
its available in Message interface which is being extended here.